module_bindings/dripline_core/run_simple_service_pybind.hh

```diff
@@ -0,0 +1,12 @@
+#include "run_simple_service.hh"
+#include "pybind11/pybind11.h"
+
+namespace dripline_pybind {
+
+  void export_run_simple_service(pybind11::module& mod){
```

Comment:
same comment for `{}` blocks

our convention for arguments is to include a space after the open and before the close, so:
`void export_run_simple_service( pybind11::module& mod )`